pkgs/shells/fish/plugins/fish-you-should-use.nix

```diff
@@ -0,0 +1,26 @@
+{
+  lib,
+  buildFishPlugin,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+buildFishPlugin {
+  pname = "fish-you-should-use";
+  version = "0-unstable-2022-02-13";
+
+  src = fetchFromGitHub {
+    owner = "paysonwallach";
+    repo = "fish-you-should-use";
+    rev = "a332823512c0b51e71516ebb8341db0528c87926";
+    hash = "sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=";
```

Comment:
```suggestion
    hash = "sha256-MmGDFTgxEFgHdX95OjH3jKsVG1hdwo6bRht+Lvvqe5Y=";
```

Reply:
I didn't pay attention to the hash mismatch when I previously checked this. Thanks!

Reply:
Sorry for not noticing that. Thanks for catching it!